Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+| Cores TRL Target identified	                      | Y                               |                                                                               |
+| Project release plan identified                     | N                               | Process to define in the main project. Plans to define in the sub-projects    |
+| HL Project deliverables identified                  | Y                               |                                                                               |
+| Feature list available                              | Y                               | Requirement specification                                                     |
+| Resource plan available                             | Y                               | Team in place for the main project. Sub-projects to identify their resources. |
+| Repo setup                                          | Y                               |                                                                               |
+| License.md file in place                            | Y                               |                                                                               |
+| Project Manager identified                          | Y                               |                                                                               |
+| Technical Project Leader per deliverable identified | Y                               | For the main project                                                          |
+| At least 1 project committer elected                | Y                               |                                                                               |
+| Work Breakdown Structure available                  | Y                               |                                                                               |
+| Baseline schedule available                         | Y                               |                                                                               |
+| Ongoing schedule tracking identified                | Y                               |                                                                               |
+| Regular project meeting setup                       | Y                               |                                                                               |
+| Project Monthly report format agreed                | Y                               |                                                                               |
+| Risk Register available                             | Y                               |                                                                               |
+| Set of Project Freeze/Release Checklists identified | N                               | To be defined in sub-projects                                                 |
+
```

Comment:
Marketing need was mentioned but few specific points were mentioned. Given the scope of this project (a super-project!) this is quite important. Let's discuss some specifics and try to capture them in this doc.